### ISSUE TITLE
Implement from foo import *

### ISSUE THREAD

**Issue by csvoss** [csvoss/oneliner#47]

**Comment by mitchellrj** [comment]
`from module import *` could be implemented as `locals().update(vars(__import__('module')))`﻿

**Comment by andersk** [comment]
Not quite, both because we don’t use the real `locals()` and because `*` must only give you [public names](https://docs.python.org/2.7/reference/simple_stmts.html#the-import-statement) from the module.
